app/models/like.rb

```diff
@@ -0,0 +1,12 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
```

Comment:
The foreign key is improperly implemented. As per the given ERD, the Like association should have Like belong to User through Author, an alias of User, like below:
`belongs_to :author, class_name: 'User'`

Kindly ensure to improve upon this aspect.

Reply:
Hi @petudeveloper I do not think I need to use class_name: 'User' as the user_id is the foreign key I am using and that will automatically match the User class